doc/api/modules.md

```diff
@@ -529,34 +529,43 @@ const x = require('./x');
 console.log(x.a);
 ```
 
-#### exports alias
+#### exports shortcut
 <!-- YAML
 added: v0.1.16
 -->
 
-The `exports` variable that is available within a module starts as a reference
-to `module.exports`. As with any variable, if you assign a new value to it, it
-is no longer bound to the previous value.
+The `exports` variable that is available within a module starts as if
+`module.exports` had been assigned to it before your module was evaluated.
+
+It allows a shortcut, so that `module.exports.f = ...` can be written more
+succinctly as `exports.f = ...`. However, be aware that like any variable, if
+you assign a new value to it, it is no longer bound to `module.exports`:
```

Comment:
For example, this last sentence is better written as: "Note, however, assigning a new value to `exports` causes it to no longer be bound to `module.exports`."